x-pack/examples/ui_actions_enhanced_examples/README.md

```diff
@@ -1,3 +1,25 @@
-## Ui actions enhanced examples
+# Ui actions enhanced examples
 
 To run this example, use the command `yarn start --run-examples`.
+
```

Comment:
I think it would be a good idea to provide a little description of how to view each demo. Where to click, what to do, etc.

Reply:
I've added description to `README.md` about what user needs to do and click to find drilldowns.